openlibrary/components/MergeUI.vue

```diff
@@ -89,21 +110,34 @@ export default {
     margin-bottom: 5px;
     padding: 5px;
 
-    .merge-btn {
-        background-color: green;
+    & > button {
+        font-size: 1.3em;
+        padding: 10px;
+        margin: 5px;
+        border: none;
+        border-radius: 5px;
         color: white;
     }
 
+    .merge-btn {
+        background-color: rgb(76, 118, 76);
+    }
+    .merge-btn:hover {
+        background-color: rgb(100, 156, 100);
+    }
+
```

Comment:
Make it look more disabled when disabled
```suggestion

    .merge-btn[disabled] {
        background-color: rgb(117, 117, 117);
    }
    .merge-btn[disabled]:hover {
        background-color: rgb(117, 117, 117);
    }
```